src/aardwolf/models/account.rs

```diff
@@ -4,6 +4,7 @@ use chrono::NaiveDateTime;
 use schema::aardwolf::accounts;
 
 #[derive(Queryable, Identifiable)]
+#[allow(unused_must_use, dead_code)]
```

Comment:
I would prefer to avoid over-riding the toolchain's default behaviors. Is there a particular reason that we're doing so here, or is it "just to get rid of those warnings"?

Reply:
fixed!